minecraftd-init@.service.in

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
I tried everything, but unfortunately because systemd's sandboxing affects all `ExecStartPre=` commands too, there was no way to achieve full and proper isolation without a separate unit.

Reply:
Personally, I would much rather print an error and require user intervention at this point than have a systemd service lying around that runs exactly once.